Library/OcDevicePathLib/OcDevicePathLib.c

```diff
@@ -1118,10 +1121,6 @@ OcFileDevicePathFullName (
   ASSERT (IsDevicePathValid (&FilePath->Header, 0));
   ASSERT (PathNameSize == OcFileDevicePathFullNameSize (&FilePath->Header));
 
-  //
-  // FIXME: Insert separators between nodes if not present already.
-  //
-
```

Comment:
Based on MdePkg DevicePathToText.c https://github.com/acidanthera/audk/blob/master/MdePkg/Library/UefiDevicePathLib/DevicePathToText.c#L2433 I believe it is reasonable to assume that all nodes always need a separator.

Reply:
The spec says they are optional, on both ends actually. I don't understand why this garbage is supported anyway, but should maybe leave a note about it?

Reply:
I'll add back in a note saying that it's supposedly possible but we haven't seen it (and is not currently supported here), at the place of this removed comment.

I'll hold off a sec, until we've thrashed out the previous point!

Reply:
@mhaeuser Do you have a link to the documentation (quick search has not yet turned it up for me)? Thank you.

Reply:
![image](https://user-images.githubusercontent.com/8659494/118269182-b184fa00-b4be-11eb-8dd0-90a13f4a4939.png)